background.ts

```diff
@@ -388,45 +388,51 @@ Electron.ipcMain.on('k8s-state', (event) => {
   event.returnValue = k8smanager.state;
 });
 
-Electron.ipcMain.on('k8s-reset', async(event, arg) => {
-  try {
-    // If not in a place to restart than skip it
-    if (![K8s.State.STARTED, K8s.State.STOPPED, K8s.State.ERROR].includes(k8smanager.state)) {
-      console.log(`Skipping reset, invalid state ${ k8smanager.state }`);
-
-      return;
-    }
-
-    if (k8smanager.version !== cfg.kubernetes.version ||
-      (await k8smanager.cpus) !== cfg.kubernetes.numberCPUs ||
-      (await k8smanager.memory) !== cfg.kubernetes.memoryInGB * 1024) {
-      arg = 'slow';
-    }
-    switch (arg) {
-    case 'fast':
-      await k8smanager.reset(cfg.kubernetes);
-      break;
-    case 'slow': {
-      await k8smanager.stop();
-
-      console.log(`Stopped Kubernetes backened cleanly.`);
-      console.log('Deleting VM to reset...');
-      await k8smanager.del();
-      console.log(`Deleted VM to reset exited cleanly.`);
+Electron.ipcMain.on('current-port', (event) => {
```

Comment:
Cross-process things should be async:
```suggestion
Electron.ipcMain.handle('current-port', () => k8smanager.port);
```

Reply:
Done in 424da72